### PR TITLE
fix typo for Windows VS build path

### PR DESCRIPTION
**Pull request by jeremiedb**

### CODE REVIEW

File: R-package/src/install.libs.R [hunk 1]
```diff
@@ -46,7 +46,7 @@ if (!use_precompile) {
     } else {
       cmake_cmd <- paste0(cmake_cmd, " -DCMAKE_GENERATOR_PLATFORM=x64 ")
       build_cmd <- "cmake --build . --target _lightgbm  --config Release"
-      lib_folder <- paste0(R_PACKAGE_SOURCE, "src/Release/")
```
Comment (guolinke) [inline R-package/src/install.libs.R]: I don't think this is a typo. I remember R_PACKAGE_SOURCE is end with ``/`` .

**Comment by Laurae2** [comment]
There is already a slash on R_PACKAGE_SOURCE.

Does the omission of the double slash fixes an error?

**Comment by jeremiedb** [comment]
I noticed other paths seem to assume no slash at the end of R_PACKAGE_SOURCE, for example at line 10: 
`source_dir <- paste0(R_PACKAGE_SOURCE, "/src/")`

My installation on Windows 10 with R 3.4.0 patched failed without this slash: failed to find the DLL. 

I also noticed another bug in that install, trying to run the agaricus example results in: 
```
Error in lgb.call("LGBM_DatasetCreateFromCSC_R", ret = handle, private$raw_data@p,  : 
  api error: Number of classes must be 1 for non-multiclass training
```

**Comment by Laurae2** [comment]
@jeremiedb Can you use `file.path` instead of `paste0`?

**Comment by jeremiedb** [comment]
If I replace line 49 with: 
`lib_folder <- file.path(R_PACKAGE_SOURCE, "src/Release/")`
The compilation also fails: `Error in eval(ei, envir) : Cannot find lib_lightgbm.dll`

**Comment by guolinke** [comment]
@jeremiedb 
I think a better solution is checking ```R_PACKAGE_SOURCE``` is end with ```/``` or not .
if it is not end with ```/```, append ```/``` .

**Comment by rivershadowy** [comment]
@jeremiedb Thank you very much, just to say, your post saved me hours. :)
Same here with windows 10 and R 3.4.0. if I don't have the slash, installation fails with error: unable to find lib_lightgbm.dll.

**Comment by odiseo1982** [comment]
@rivershadowy I'm trying to install LightGBM from RStudio using

devtools::install_github("Microsoft/LightGBM", subdir = "R-package")

but I get Error in eval(ei, envir) : Cannot find lib_lightgbm.dll. I have Windows 10 and Microsoft R Open 3.4.0. Can you please tell me what did you do to install it? Thanks

**Comment by guolinke** [comment]
@odiseo1982 
can you paste the full log?

**Pull request merged**